src/lib/server/cart.ts

```diff
@@ -64,14 +64,21 @@ export async function addToCartInDb(
 		customPrice?: { amount: number; currency: Currency };
 		deposit?: boolean;
 		chosenVariations?: Record<string, string>;
+		mode?: 'eshop' | 'nostr' | 'pos';
```

Comment:
can you make this parameter mandatory?

```suggestion
		mode: 'eshop' | 'nostr' | 'pos';
```

you can use Ctrl+Shift+F in VS code or just shift+F12 when on the function to find all references.

Currently all references have the "mode" added.So it can be made mandatory